clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -446,6 +446,8 @@ def err_drv_print_header_env_var_combination_cc1 : Error<
   "unsupported combination: -header-include-format=%0 and -header-include-filtering=%1">;
 
 def warn_O4_is_O3 : Warning<"-O4 is equivalent to -O3">, InGroup<Deprecated>;
+def warn_invalid_fp64_emu_use : Warning<"Invalid option for %0; -fsycl-fp64-conv-emu option is not supported for non-Intel GPU or JIT compilation">,
```

Comment:
```suggestion
def warn_invalid_fp64_emu_use : Warning<"Invalid option for target %0; '-fsycl-fp64-conv-emu' is not supported for non-Intel GPU or JIT compilation">,
```
Alternatively:  `Invalid option for target %0; '-fsycl-fp64-conv-emu' is only supported for Intel GPUs`